### PR TITLE
Manage editable vs editable[col] (#464)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
#465 Changed the doc for editable default to True

### CODE REVIEW

File: gui/doc/table.csv [hunk 1]
```diff
@@ -34,7 +34,7 @@ filter,bool,FALSE,"Indicates, if True, that all columns can be filtered."
 filter[<i>col_name</i>],bool,FALSE,"Indicates, if True, that the indicated column can be filtered."
 nan_value,str,"""""",The replacement text for NaN (not-a-number) values.
 nan_value[<i>col_name</i>],str,"""""",The replacement text for NaN (not-a-number) values for the indicated column.
-editable,dynamic(bool),FALSE,"Indicates, if True, that all columns can be edited."
+editable,dynamic(bool),True,"Indicates, if True, that all columns can be edited."
```
Comment (FabienLelaquais) [inline gui/doc/table.csv]: Please make sure that this is ported back to viselements.json...

File: gui/doc/table.csv [hunk 2]
```diff
@@ -81,6 +81,6 @@ This dictionary has the following keys:
 </li>
 </ul>
 If this property is not set, the user cannot add rows."
-size,str,"""small""","The size of the rows, valid values are small or medium."
+size,str,"""small""","The size of the rows, valid values are small and medium."
```
Comment (FabienLelaquais) [inline gui/doc/table.csv]: valid values are "small" and "medium"

**Review by FabienLelaquais — APPROVED** [review]
👌

File: gui/doc/table.csv (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/doc/table.csv]: it'll be in my next PR.

File: gui/doc/table.csv (hunk 2 above)
Comment (FredLL-Avaiga) [reply gui/doc/table.csv]: modified in viselements.json

**Pull request merged**